### PR TITLE
Development

### PR DESCRIPTION
**Pull request by hannasoderstrom1**
Uses Alissa_API_tools as a submodule to upload germline SNV_CNV vcf to Alissa

Uses /medstore/results/wgs/wgs_somatic as workingdir and copies resultfiles to /seqstore/webfolders after pipeline has finished

Copies files to IGV instead of symlink and also copies to new IGV dir on webstore

### CODE REVIEW

File: workflows/rules/results_sharing/share_to_igv.smk [hunk 1]
```diff
@@ -21,10 +22,10 @@ if tumorid:
                 expand("{workingdir}/{stype}/manta/{sname}_somatic_MantaNOBNDs.vcf", workingdir=workingdir, sname=tumorid, stype=sampleconfig[tumorname]["stype"]),
                 expand("{workingdir}/{stype}/manta/{sname}_germline_MantaBNDs.vcf", workingdir=workingdir, sname=normalid, stype=sampleconfig[normalname]["stype"]),
                 expand("{workingdir}/{stype}/manta/{sname}_germline_MantaNOBNDs.vcf", workingdir=workingdir, sname=normalid, stype=sampleconfig[normalname]["stype"]),
-                expand("{workingdir}/{stype}/tnscope/{sname}_REALIGNED_realignedTNscope.bam", workingdir=workingdir, sname=tumorid, stype=sampleconfig[tumorname]["stype"]),
+                #expand("{workingdir}/{stype}/tnscope/{sname}_REALIGNED_realignedTNscope.bam", workingdir=workingdir, sname=tumorid, stype=sampleconfig[tumorname]["stype"]),
```
Comment (alvaralmstedt) [inline workflows/rules/results_sharing/share_to_igv.smk:25]: why

File: workflows/rules/results_sharing/share_to_igv.smk [hunk 2]
```diff
@@ -21,10 +22,10 @@ if tumorid:
...
                 expand("{workingdir}/{stype}/pindel/{sname}_pindel.vcf", workingdir=workingdir, sname=tumorid, stype=sampleconfig[tumorname]["stype"])
             params:
-                updateigv = pipeconfig["rules"]["share_to_igv"]["updateigv"],
+                #updateigv = pipeconfig["rules"]["share_to_igv"]["updateigv"],
```
Comment (alvaralmstedt) [inline workflows/rules/results_sharing/share_to_igv.smk:28]: why

File: workflows/rules/results_sharing/share_to_igv.smk [hunk 3]
```diff
@@ -77,16 +81,18 @@ if tumorid:
                         shell(f"{params.bcftools} index -f {sharefile}.gz")
                         sharefile = os.path.abspath(f"{sharefile}.gz")
                     link_sharefile = os.path.abspath(sharefile)
-                    shell("ln -sf {link_sharefile} {igvsharedir}")
+                    #shell("ln -sf {link_sharefile} {igvsharedir}")
```
Comment (alvaralmstedt) [inline workflows/rules/results_sharing/share_to_igv.smk:84]: why

File: README.md [hunk 4]
```diff
@@ -97,6 +99,18 @@ Need to be on medair because python environment is hard-coded in launch_snakemak
  * Robust and well-documented to fit into the clinical requirements.
 
 
+ ### Yearly statistics
+
+After running the pipeline for the first time, a yearly\_statistics text file is created in the repo. Every time the pipeline is run, sample name and date/time is added to this text file.
```
[the remark this answers is not in the document]
Comment (hannasoderstrom1) [reply README.md:104]: yes

File: workflows/rules/results_sharing/share_to_igv.smk (hunk 1 above)
Comment (hannasoderstrom1) [reply workflows/rules/results_sharing/share_to_igv.smk:25]: don't think they look at it in igv. if they complain, i'll add it again

File: workflows/rules/results_sharing/share_to_igv.smk (hunk 2 above)
Comment (hannasoderstrom1) [reply workflows/rules/results_sharing/share_to_igv.smk:28]: updates automatically every 15 mins anyway

**Pull request merged**